Simplified/Book/Models/NYPLBookRegistry.m

```diff
@@ -310,98 +310,97 @@ - (void)syncResettingCache:(BOOL)shouldResetCache
     [self broadcastChange];
   } //@synchronized
   
-  [NYPLOPDSFeed
-   withURL:[[[AccountsManager sharedInstance] currentAccount] loansUrl]
-   shouldResetCache:shouldResetCache
-   completionHandler:^(NYPLOPDSFeed *const feed, NSDictionary *error) {
-     if(!feed) {
-       NYPLLOG(@"Failed to obtain sync data.");
-       self.syncing = NO;
-       [self broadcastChange];
-       [[NSOperationQueue mainQueue]
-        addOperationWithBlock:^{
-          if(completion) completion(error);
-          if(fetchHandler) fetchHandler(UIBackgroundFetchResultFailed);
-          [[NSNotificationCenter defaultCenter] postNotificationName:NSNotification.NYPLSyncEnded object:nil];
-        }];
-       [NYPLErrorLogger logErrorWithCode:NYPLErrorCodeApiCall
-                                 summary:@"Unable to fetch loans"
-                                metadata:@{
-                                  @"shouldResetCache": @(shouldResetCache),
-                                  @"errorDict": error ?: @"N/A"
-                                }];
-       return;
-     }
+  [NYPLOPDSFeedFetcher fetchOPDSFeedWithUrl:[[[AccountsManager sharedInstance] currentAccount] loansUrl]
+                           shouldResetCache:shouldResetCache
+                                 completion:^(NYPLOPDSFeed * _Nullable feed, NSDictionary<NSString *,id> * _Nullable errorDict) {
```

Comment:
The body of this completion block contains no changes except the name of the parameters and indentation.